WordPress/src/main/java/org/wordpress/android/ui/posts/PostListItemViewHolder.kt

```diff
@@ -31,13 +31,13 @@ class PostListItemViewHolder(
     private val config: PostViewHolderConfig,
     private val uiHelpers: UiHelpers
 ) : RecyclerView.ViewHolder(LayoutInflater.from(parentView.context).inflate(layout, parentView, false)) {
-    private val ivImageFeatured: ImageView = itemView.findViewById(R.id.image_featured)
-    private val tvTitle: WPTextView = itemView.findViewById(R.id.title)
-    private val tvExcerpt: WPTextView = itemView.findViewById(R.id.excerpt)
-    private val tvDateAndAuthor: WPTextView = itemView.findViewById(R.id.date_and_author)
-    private val tvStatusLabels: WPTextView = itemView.findViewById(R.id.status_labels)
-    private val pbProgress: ProgressBar = itemView.findViewById(R.id.progress)
-    private val flDisabledOverlay: FrameLayout = itemView.findViewById(R.id.disabled_overlay)
+    private val featuredImageView: ImageView = itemView.findViewById(R.id.image_featured)
+    private val titleTextView: WPTextView = itemView.findViewById(R.id.title)
+    private val excerptTextView: WPTextView = itemView.findViewById(R.id.excerpt)
+    private val dateAndAuthorTextView: WPTextView = itemView.findViewById(R.id.date_and_author)
+    private val statusesLabelTextView: WPTextView = itemView.findViewById(R.id.statuses_label)
+    private val progressProgressBar: ProgressBar = itemView.findViewById(R.id.progress)
```

Comment:
ℹ️ This seems a bit generic. Does `uploadProgressBar` sound and explain its purpose better? 🤔

Reply:
Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/9514/commits/092f3d73eec6d6aaeba64569f78ea1868dfee643